cxx4opencl/diff2openclc.txt

```diff
@@ -129,6 +129,18 @@ label:  // invalid: jumping forward over declaration of n
    // ...
 ----------
 
+===== Ternary Selection Operator
+
+The ternary selection operator (`?:`) inherits its behaviour from both
+{cpp} and OpenCL C. It operates on three expressions
+`(exp1 ? exp2 : exp3)`. If all three expressions are scalar values,
+the {cpp}17 rules for ternary operator are followed. If the result is
```

Comment:
I am not sure if `the result` is clearly defined in this context. Maybe it can be explained before this sentence?

Reply:
Although I appreciate that OpenCL C wording is the same...

Reply:
If I remember correctly, using `the result` is the most precise about the actual implementation inside clang.

Reply:
ok, but the language documentation should be agnostic to the exact compiler. We can document clang specific aspects in the clang internal documentation... However,  here I don't think the result is clear here. I would at least change this to something like the expression result or the operator result?